website/docs/monorepo.md

```diff
@@ -0,0 +1,97 @@
+---
+id: monorepo
+title: Monorepo
+---
+
+In this guide, we will configure Docusaurus for a Monorepo.
+Our configuration will enable the following directory structure.
+```
+. (project root)
+|_ docs (this is where the Monorepo's main docs go)
```

Comment:
The lines don't look very good. Recommend using the ones on https://v2.docusaurus.io/docs/installation#project-structure instead.